ci-operator/config/openshift/azure-disk-csi-driver-operator/openshift-azure-disk-csi-driver-operator-master.yaml

```diff
@@ -50,6 +50,10 @@ tests:
   steps:
     cluster_profile: azure4
     workflow: openshift-e2e-azure-csi
+- as: e2e-azure-csi-migration
+  steps:
+    cluster_profile: azure4
+    workflow: openshift-e2e-azure-csi-migration
```

Comment:
Please add this change also to `openshift-azure-disk-csi-driver-operator-4.9.yaml` and `release/ci-operator/config/openshift/azure-disk-csi-driver/openshift-azure-disk-csi-driver-{master,4.9}.yaml

Reply:
@jsafrane  done